californium-core/src/main/java/org/eclipse/californium/core/network/config/NetworkConfigDefaults.java

```diff
@@ -85,6 +85,7 @@ public static void setDefaults(final NetworkConfig config) {
 		config.setFloat(NetworkConfig.Keys.PROBING_RATE, 1f);
 
 		config.setBoolean(NetworkConfig.Keys.USE_RANDOM_MID_START, true);
+		config.setInt(NetworkConfig.Keys.MID_PROVIDER_GROUPS, 16);
```

Comment:
can we define a constant for the default value? makes lookup using JavaDocs much, much easier for users.